Resources/Prototypes/_Mono/Entities/SpaceArtillery/Ammo/50cal.yml

```
@@ -50,28 +50,28 @@
+  parent: 57mmBulletBase
   categories: [ HideSpawnMenu ]
   components:
   - type: Projectile
     damage:
       types:
-        Structural: 95
+        Structural: 15
   - type: Sprite
     sprite: _Mono/Objects/SpaceArtillery/50_armorpiercing_machinegun_casing.rsi
     layers:
     - state: base-projectile
```

YAMLlint (1.35.1)

[error] wrong indentation: expected 6 but found 4

(indentation)